enterprise-suite/templates/es-console-configmap.yaml

```diff
@@ -206,16 +212,28 @@ data:
         etag off;
       }
 
+      # Specific handler for assets.
+      location ~ ^(.*)(/assets/.*)$ {
+        alias $base$2;
+      }
 
-      # static files
+      # Handler for application routes and assets not inside of /assets.
+      location ~ ^(.*)(/(cluster|.*\.js|.*\.css|.*\.woff|.*\.woff2|.*\.ttf))$ {
```

Comment:
the "/cluster" route needs to be handled like the other application routes or else it won't have the right headers, cache control, security handlers, etc. Unfortunately I think we need another copy / paste section of the chunky section copy pasted in this config...

Reply:
I don't think any of this has changed - do you mean it's already broken? This should rewrite `/cluster` to the `index.html` location block which will provide the correct headers.

Reply:
It looks okay to me - please clarify:

```
$ curl -sD - http://192.168.99.116:30080/cluster
HTTP/1.1 200 OK
Date: Mon, 25 Feb 2019 09:55:04 GMT
Content-Type: text/html
Transfer-Encoding: chunked
Connection: keep-alive
X-Frame-Options: DENY
X-XSS-Protection: 1
Content-Security-Policy: default-src 'self' 'unsafe-eval' 'unsafe-inline';
Cache-Control: no-store, no-cache, must-revalidate, proxy-revalidate, max-age=0
```

Reply:
Ah right this will flow into the index.html handler. Now I'm wondering if we can do the same for the copy/paste sections (next time)